LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017 Braintree, a division of PayPal, Inc.
+Copyright (c) 2017-2020 Braintree, a division of PayPal, Inc.
```

Comment:
Does this need to be updated? I don't think you need to include the current year here.

Reply:
```suggestion
Copyright (c) 2017 Braintree, a division of PayPal, Inc.
```